llama_stack/apis/telemetry/telemetry.py

```diff
@@ -123,10 +137,67 @@ class StructuredLogEvent(EventCommon):
 ]
 
 
+@json_schema_type
+class EvalTrace(BaseModel):
+    session_id: str
+    step: str
+    input: str
+    output: str
+    expected_output: str
+
+
+@json_schema_type
+class MaterializedSpan(Span):
+    children: List["MaterializedSpan"] = Field(default_factory=list)
+    status: Optional[SpanStatus] = None
+
+
+@json_schema_type
+class QueryCondition(BaseModel):
+    key: str
+    op: str
```

Comment:
make it a simple enum like `"eq" | "ne" | "gt" | "lt"`

Reply:
done.